src/tyr/route_summary_cache.h

```diff
@@ -1,3 +1,5 @@
+#ifndef VALHALLA_TYR_ROUTE_SUMMARY_CACHE_H_
+#define VALHALLA_TYR_ROUTE_SUMMARY_CACHE_H_
 #pragma once
```

Comment:
shouldn't `pragma once` be enough here? I thought it's properly supported by most toolchains by now?

Reply:
`pragma once` is not standard